tests/shared/permissions.spec.js

```diff
@@ -76,11 +76,11 @@ describe('Shared: permissions', function() {
       it('should ignore undefined and null variable values', function() {
         m.chai.expect(permissions.getEnvironmentCommandPrefix({
           FOO: null,
-          BAR: undefined,
-          BAZ: 'qux'
+          BAR: 'qux',
+          BAZ: undefined
```

Comment:
For consistency, this change of ordering should be done for the other 2 OSes too!